volumes/miovision/sql/function/function-clear-api_log.sql

```diff
@@ -0,0 +1,31 @@
+﻿CREATE OR REPLACE FUNCTION miovision_api.clear_api_log(
+    _start_date date,
+    _end_date date,
+    intersection integer DEFAULT NULL)
+RETURNS void
```

Comment:
not like an important feature but it will be helpful if we can see how many rows got deleted, or if anything got deleted at all